File: Stripe/STPPaymentMethodPrzelewy24Params.m
//
//  STPPaymentMethodPrzelewy24Params.m
//  StripeiOS
//
//  Created by Vineet Shah on 4/23/20.
//  Copyright © 2020 Stripe, Inc. All rights reserved.
//

#import "STPPaymentMethodPrzelewy24Params.h"

NS_ASSUME_NONNULL_BEGIN

@implementation STPPaymentMethodPrzelewy24Params

@synthesize additionalAPIParameters = _additionalAPIParameters;

+ (nullable NSString *)rootObjectName {
    return @"p24";
}

+ (NSDictionary *)propertyNamesToFormFieldNamesMapping {
    return @{
             };
}

@end

NS_ASSUME_NONNULL_END

File: StripePayments/StripePayments/API Bindings/Models/PaymentMethods/Types/STPPaymentMethodPrzelewy24Params.swift
//
//  STPPaymentMethodPrzelewy24Params.swift
//  StripePayments
//
//  Created by Vineet Shah on 4/23/20.
//  Copyright © 2020 Stripe, Inc. All rights reserved.
//

import Foundation

/// An object representing parameters used to create a Przelewy24 Payment Method
public class STPPaymentMethodPrzelewy24Params: NSObject, STPFormEncodable {
    @objc public var additionalAPIParameters: [AnyHashable: Any] = [:]

    @objc
    public class func rootObjectName() -> String? {
        return "p24"
    }

    @objc
    public class func propertyNamesToFormFieldNamesMapping() -> [String: String] {
        return [:]
    }
}
